code/modules/Economy/Accounts.dm

```diff
@@ -33,14 +34,16 @@ var/global/list/all_money_accounts = list()
 		station_account.transaction_log.Add(T)
 		all_money_accounts.Add(station_account)
 
-/proc/create_department_account(department)
+/proc/create_department_account(department, var/recieves_wage = 0)
 	next_account_number = rand(111111, 999999)
 
 	var/datum/money_account/department_account = new()
 	department_account.owner_name = "[department] Account"
 	department_account.account_number = rand(11111, 99999)
 	department_account.remote_access_pin = rand(1111, 9999)
 	department_account.money = 5000
+	if(recieves_wage == 1)
+		department_account.wage_gain = 500
```

Comment:
Define this.

The fact that 5000 is repeated 3 times without a define is disgusting.